lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -42,6 +42,12 @@ function subscribeEvents() {
   });
 }
 
+export function sendEvent(category, action, label) {
+  if (typeof ga !== 'undefined' && ga !== null) {
+    ga('send', 'event', category, action, label);
+  }
+}
+
 function init() {
   // Only fire GA Custom Events if the GA object exists.
   if (typeof ga !== 'undefined' && ga !== null) {
```

Comment:
if definitely needed in `sendEvent()` since it's an exported function, then maybe not needed here, since the init calls that function and the check is done there?